60-suspend-mode.rules

```diff
@@ -0,0 +1 @@
+SUBSYSTEM=="dmi", ENV{ID_VENDOR}="GIGABYTE", ENV{ID_MODEL}="Mission one", RUN{program}+="/bin/sh -c 'echo s2idle > /sys/power/mem_sleep'"
```

Comment:
From https://phabricator.endlessm.com/w/hardware/endless/mission-one/ I would have guessed that it is `GB-BXBT-2807` but it's true that https://github.com/endlessm/systemd/commit/1ecc9fe237a750a65249b948f4b28834a6c1fc16 uses `Mission one` (as well as `GB-BXBT-2807`).

I updated https://metabase.endlessm.com/question/267-pings-from-mission-one to look at both names, and `GB-BXBT-2807` way outnumbers `Mission one`. I think we should treat both the same, just as the old patch does.

Reply:
Do they have the same sys_vendor "GIGABYTE"?

Reply:
Pushed a new commit.  Ready for review again.